content/blog/2017/0.1-canary/index.md

```diff
@@ -34,8 +34,8 @@ Whether we use one deployment or two, canary management using deployment feature
 
 With Istio, traffic routing and replica deployment are two completely independent functions. The number of pods implementing services are free to scale up and down based on traffic load, completely orthogonal to the control of version traffic routing. This makes managing a canary version in the presence of autoscaling a much simpler problem. Autoscalers may, in fact, respond to load variations resulting from traffic routing changes, but they are nevertheless functioning independently and no differently than when loads change for other reasons.
 
-Istio’s [routing rules](/docs/concepts/traffic-management/#rule-configuration) also provide other important advantages; you can easily control
-fine grain traffic percentages (e.g., route 1% of traffic without requiring 100 pods) and you can control traffic using other criteria (e.g., route traffic for specific users to the canary version). To illustrate, let’s look at deploying the **helloworld** service and see how simple the problem becomes.
+Istio’s [routing rules](/docs/concepts/traffic-management/routing/virtual-services/#routing-rules) also provide other important advantages; you can easily control
```

Comment:
you should not call this individual piece "/docs/concepts/traffic-management/routing/virtual-services/#routing-rules" as routing rules.. While its called so in the API, people usually talk about the entire gamut of routing in Istio as routing rules. So I suggest rephrasing this piece as 
```
Traffic routing in Istio also provide other ..
```